knimin/lib/tests/test_data_access.py

```diff
@@ -1408,6 +1447,48 @@ def test_delete_pool(self):
             db.delete_pool(0)
         self.assertEqual(ctx.exception.message, "Pool 0 does not exist")
 
+    def test_get_pool_list(self):
+        self.assertEqual(db.get_pool_list(), [])
+
+        pool_ids = self._create_test_data_pool()
+        exp = [{'id': pool_ids[0], 'name': 'LabAdmin test pool',
+                'targeted_pools': ['Test plate']},
+               {'id': pool_ids[1], 'name': 'LabAdmin test pool 2',
+                'targeted_pools': ['Test plate 2']}]
+
+        self.assertEqual(db.get_pool_list(), exp)
+
+    def test_create_sequencing_run(self):
+        pool_ids = self._create_test_data_pool()
+
+        # Create the run
+        before = datetime.datetime.now()
+        run_id = db.create_sequencing_run(
+            pool_ids[0], 'test', 1, 'MiSeq v3 150 cycle', 'MS1234')
+        after = datetime.datetime.now()
+        self._clean_up_funcs.insert(
+            0, partial(db.delete_sequencing_run, run_id))
+
+        obs = db.read_sequencing_run(run_id)
+        self.assertTrue(before <= obs.pop('created_on') <= after)
```

Comment:
I wonder if this will always pass. Is it possible that Python and Postgres use two different ways to get current time, and they may differ by a few minisecs?